release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -950,7 +950,7 @@ New enhancements are available on the *Monitoring* -> *Dashboards* page in the {
 
 [id="ocp-4-8-metering"]
 === Metering
-The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled to be removed in {product-title} 4.9.
+The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled to be removed in {product-title} next release.
```

Comment:
nit:

> ... removed in OpenShift Container Platform next release.

reads a bit strangely to me.  I'd have expected:

> ... removed in the next release of OpenShift Container Platform.

or some such.

Reply:
+1 I'd suggest either the wording that @wking proposed, or "the next {product-title} release"